frontend/src/pages/Support/Support.test.tsx

```diff
@@ -0,0 +1,135 @@
+import { Calendar } from 'antd';
+import { Book, Cable, Github, MessageSquare, Slack } from 'lucide-react';
+import { fireEvent, render } from 'tests/test-utils';
+
+import Support from './Support';
+
+const launchChat = 'Launch chat';
+const useAnalyticsMock = jest.fn();
+const useHistoryMock = jest.fn();
+
+jest.mock('hooks/analytics/useAnalytics', () => ({
+	__esModule: true,
+	default: jest.fn(() => ({ trackEvent: useAnalyticsMock })),
+}));
+
+jest.mock('react-router-dom', () => ({
+	useHistory: useHistoryMock,
+}));
+
+const supportChannels = [
+	{
+		key: 'documentation',
```

Comment:
we already have the support channels in the code right ? please export it from there and use here. this increases the maintainability of the code and helps keep the code DRY